packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -202,17 +204,15 @@ class InteractiveGraphEditor extends React.Component<Props> {
             Util.getGridStep(
                 this.props.range,
                 this.props.step,
-                interactiveSizes.defaultBoxSize,
```

Comment:
This actually results in the same grid step either way, but I changed it to the small one for consistency.